src/extract.cpp

```diff
@@ -359,27 +351,24 @@ bool my_plugin::extract(const falcosecurity::extract_fields_input& in) {
             SPDLOG_DEBUG("the plugin has no container_id info for the thread id '{}'", thread_id);
             return false;
         }
-    }
-    catch(falcosecurity::plugin_exception e)
-    {
-        SPDLOG_ERROR("cannot extract the container_id for the thread id '{}': {}",
+    } catch(falcosecurity::plugin_exception e) {
+      	// Debug here since many events do not have thread id info (eg: schedswitch)
+        SPDLOG_DEBUG("cannot extract the container_id for the thread id '{}': {}",
```

Comment:
Just a debug here: there can be events (eg: schedswitch) that do not have any threadinfo associated.